test/model/dataTypes/EntityCollection.test.ts

```diff
@@ -0,0 +1,107 @@
+import DataType from "../../../src/model/dataTypes/DataType";
+import EntityCollection from "../../../src/model/dataTypes/EntityCollection";
+import EntityDefinition from "../../../src/model/dataTypes/EntityDefinition";
+import EntityMetadata from "../../../src/model/dataTypes/EntityMetadata";
+import EntityImports from "../../../src/model/dataTypes/EntityImports";
+import FilePath from "../../../src/model/dataTypes/FilePath";
+import ModelInfo from "../../../src/model/modelInfo/ModelInfo";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing file extension for "../../../src/model/modelInfo/ModelInfo"](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5691315)